### ISSUE TITLE
Python Terminal not Accessible via Screen Reader

### ISSUE THREAD

**Issue by naomi-lgbt** [freeCodeCamp/freeCodeCamp#60449]
Got this feedback via an email:

> I'm currently going through the "Scientific Computing with Python" course on freeCodeCamp, and I've encountered an accessibility issue. The terminal/output section on the coding platform is not accessible with a screen reader. Although the Python console itself is accessible, the terminal is necessary to view the output — and unfortunately, that part is not being read at all by my screen reader.

I have not attempted to reproduce this (heck, I don't even have a functional screen reader anymore). But we should definitely get eyes on this and confirm the issue + fix it.
